youtube_dl/extractor/chaturbate.py

```diff
@@ -7,6 +7,15 @@
 class ChaturbateIE(InfoExtractor):
     _VALID_URL = r'https?://(?:[^/]+\.)?chaturbate\.com/(?P<id>[^/?#]+)'
     _TESTS = [{
+        'url': 'https://www.chaturbate.com/surfergirl121/',
+        'info_dict': {
+            'id': 'surfergirl121',
+            'ext': 'mp4',
+            'title': 're:^surfergirl121 [0-9]{4}-[0-9]{2}-[0-9]{2} [0-9]{2}:[0-9]{2}$',
+            'age_limit': 18,
+            'is_live': True,
+        },
```

Comment:
I've already pointed out there is no need in another test since they are broken once room is offline.